src/AutoRest.CSharp/Common/Input/InputTypes/OperationResponse.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace AutoRest.CSharp.Common.Input;
 
-internal record OperationResponse(IReadOnlyList<int> StatusCodes, InputType? BodyType, BodyMediaType BodyMediaType, IReadOnlyList<OperationResponseHeader> Headers, bool IsErrorResponse)
+internal record OperationResponse(IReadOnlyList<int> StatusCodes, InputType? BodyType, BodyMediaType BodyMediaType, IReadOnlyList<OperationResponseHeader> Headers, bool IsErrorResponse, string[]? ContentTypes)
```

Comment:
leaning towards an empty collection
```suggestion
internal record OperationResponse(IReadOnlyList<int> StatusCodes, InputType? BodyType, BodyMediaType BodyMediaType, IReadOnlyList<OperationResponseHeader> Headers, bool IsErrorResponse, IReadOnlyList<string> ContentTypes)
```